tensorflow_lite_support/python/task/audio/core/tensor_audio.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TensorAudio class."""
+
+import numpy as np
+
+from tensorflow_lite_support.python.task.audio.core.pybinds import _pywrap_audio_buffer
+from tensorflow_lite_support.python.task.audio.core import audio_record
+
+_CppAudioBuffer = _pywrap_audio_buffer.AudioBuffer
+_CppAudioFormat = _pywrap_audio_buffer.AudioFormat
+_LoadAudioBufferFromFile = _pywrap_audio_buffer.LoadAudioBufferFromFile
+
+
+class TensorAudio(object):
+  """A wrapper class to store the input audio."""
+
+  def __init__(self,
+               audio_format: _CppAudioFormat,
+               sample_count: int,
+               audio_data: _CppAudioBuffer = None,
+               is_from_file: bool = False,
+               ) -> None:
+    """Initializes the `TensorAudio` object.
+
+    Args:
+      audio_format: C++ AudioFormat object, format of the audio.
+      sample_count: int, number of samples in the audio.
```

Comment:
There's no need to specify type hint in docstring. It can be inferred directly from the Python code.